static/javascript/index.js

```diff
@@ -55,10 +54,94 @@ function removeNoteFromDB(){
 
 }
 
+//// Reorganize Notes In Ascending/Descending Oder ////
+function upDateNoteOrderDesc(response){
+    console.log("upDateNoteOrderDesc running");
+
+    $("#contain_all_notes").empty();
+
+    
+    for(var i=0; i < response.length; i++){
+        
+                    
+        var div_create = "<div class=\"decorate_note_div\">" +
+                    "<div>" +
+                        response[i].created_at +  
+                    "</div>" +
+
+                    "<div>" +
+                        "<h4>Note Title:</h4> <p class=\"note_title\">" + response[i].title +"</p>" +
+                    "</div>" +
+
+                    "<div >" +
+                        "<h4>Note:</h4>" + 
+                        "<p class=\"notes_from_db\">" + response[i].content + "<p>" +
+                    "</div>" +
+
+                    "<button class=\"delete_note\" value=\""+ response[i].id  + "\"> Delete </button>" +
+                    "<br>" +
+                "</div>";
+         $("#contain_all_notes").append(div_create);
+        console.log(response[i])
+        console.log(div_create);
+
+
+    }
+    
+    
+}
+
+function upDateNoteOrderAscend(response){
+    console.log("upDateNoteOrderAscend running");
+
+    $("#contain_all_notes").empty();
+    
+    for(var i=0; i < response.length; i++){
+          
+        var div_create = "<div class=\"decorate_note_div\">" +
+                    "<div>" +
+                        response[i].created_at +  
+                    "</div>" +
+
+                    "<div>" +
+                        "<h4>Note Title:</h4> <p class=\"note_title\">" + response[i].title +"</p>" +
+                    "</div>" +
+
+                    "<div>" +
+                        "<h4>Note:</h4>" + 
+                        "<p class=\"notes_from_db\">" + response[i].content + "<p>" +
+                    "</div>" +
+
+                    "<button class=\"delete_note\" value=\""+ response[i].id  + "\"> Delete </button>" +
+                    "<br>" +
+                "</div>";
+        $("#contain_all_notes").append(div_create);
+        console.log(response[i])
+        console.log(div_create);
+    }
+    
+}
```

Comment:
Combine `upDateNoteOrderAscend` and `upDateNoteOrderDescend` into one function.  "update" is one word; don't capitalize the "d".